app/docs/enterprise/0.31-x/installation/docker.md

```diff
@@ -74,33 +80,37 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
 
-## FAQs
+## Enable RBAC
 
-- Starting with Kong `0.30`, the Admin API only listens on the local interface by default. This was done as a security enhancement. Note that here, we are overriding that in the above example with `KONG_ADMIN_LISTEN=0.0.0.0:8001` because Docker container networking benefits from more open settings and enables the Admin GUI & Dev Portal to talk with the Kong Proxy.
+[Role-based Access Control (RBAC)](https://getkong.org/docs/enterprise/latest/setting-up-admin-api-rbac/) allows you to create multiple Kong administrators and control which resources they have access to. To enable it:
 
-- Starting with 0.29, without a license properly referenced, you’ll get errors running migrations. Also, without a license, you'll do a “docker start <name>” and not see an error attempting to start the container. But when you check the process, it won’t be running. Doing a “docker logs <container_name>” will show you:
+1. Create an initial RBAC administrator
 
-        nginx: [alert] Error validating Kong license: license path environment variable not set
+        curl -X POST http://localhost:8001/rbac/users/ -d name=admin -d user_token=12345
+        curl -X POST http://localhost:8001/rbac/users/admin/roles -d roles=super-admin
 
-- As awareness, another error that can occur due to the vagaries of the interactions between text editors and copy & paste changing straight quotes (" or ') into curly ones (“ or ” or ’ or ‘) is:
 
-        nginx: [alert] Error validating Kong license: could not decode license json
+2. Start a bash session on the container
+        
+        docker exec -it kong-ee /bin/sh
+
+3. KONG_ENFORCE_RBAC=on kong reload
+4. Confirm that your user token is working by passing the `Kong-Admin-Token` header in requests
+        
+        curl -X GET http://localhost:8001/status -H "Kong-Admin-Token: 12345"
 
-    Your license data must contain only straight quotes to be considered valid JSON.
+If you are able to access Kong without issues, you can add `KONG_ENFORCE_RBAC=on` to your initial container environment variables.
 
-## Enable RBAC and Vitals
+## FAQs
 
-Finally, two of the key features of 0.29+ are RBAC and Vitals. To enable them:
+The Admin API only listens on the local interface by default. This was done as a security enhancement. Note that here, we are overriding that in the above example with `KONG_ADMIN_LISTEN=0.0.0.0:8001` because Docker container networking benefits from more open settings and enables the Admin GUI & Dev Portal to talk with the Kong Proxy.
 
-1. Start a bash session on the container
-        
-        docker exec -it kong-ee /bin/sh
+Without a license properly referenced, you’ll get errors running migrations. Also, without a license, you'll do a “docker start <name>” and not see an error attempting to start the container. But when you check the process, it won’t be running. Doing a “docker logs <container_name>” will show you:
```

Comment:
The docker commands here should be wrapped in backticks to be consistent with other commands and variables throughout the documentation.